doc/openapi.yaml

```diff
@@ -0,0 +1,42 @@
+swagger: "2.0"
+info:
+  version: "1.0.0"
+  title: "Sentence aligner API"
+tags:
+- name: "sentences"
+  description: "Operations on sentences"
+schemes:
+- "http"
+paths:
+  /sentences:
+    post:
+      tags:
+      - "sentences"
+      summary: "Create a new sentence."
+      consumes:
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "The sentence object that has to be created."
+        required: true
+        schema:
+          $ref: "#/definitions/Sentences"
+      responses:
+        201:
+          description: "The sentence has been created."
+        409:
+          description: "The given UUID is already used by another sentence."
+definitions:
+  Sentences:
+    type: "object"
+    properties:
```

Comment:
some fields here are required

Reply:
Are you sure we can require a field directly from the object definition ?